src/App.tsx

```diff
@@ -1,26 +1,205 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { deleteTodos, getTodos, postTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Notifications } from './components/Notifications/Notifications';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const [query, setQuery] = useState('');
+  const [filterName, setFilterName] = useState('all');
```

Comment:
```suggestion
  const [filterName, setFilterName] = useState(Filter.all);
```
The example of enum usage